index.html

```diff
@@ -154,30 +167,33 @@ <h5>LET'S CONNECT</h5>
       </div>
       <button type="button" id="aboutMyselfBotton">Get My Resume</button>
   </div>
+  <div class="desktopVersion">
   <div class="languageAndArrowContainer">
-    <span>Languages</span>
+    <span class="span-language lang">Languages</span>
     <img src="assets/downSideArrow.svg" alt="">
   </div>
   <div class="languagesContainer">
-    <ul>
-      <li class="languages"><img src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
-      <li class="languages"><img src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
-      <li class="languages"><img src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
+    <ul class="container-desktop">
+      <li class="languages"><img class="languages-img" src="assets/javascriptAsset.svg" alt="jsicon"> <span id="js">JavaScript</span></li>
+      <li class="languages"><img class="languages-img" src="assets/htmlAsset.svg" alt="htmlicon"><span id="html">HTML</span></li>
+      <li class="languages"><img class="languages-img" src="assets/cssAsset.svg" alt="cssicon"><span id="css">CSS</span></li>
     </ul>
   </div>
   <div class="languageAndArrowContainer">
-    <span>Frameworks</span>
+    <span class="span-language">Frameworks</span>
     <img src="assets/leftSideArrow.svg" alt="leftSideArrow">
   </div>
   <hr>
   <div class="languageAndArrowContainer">
-    <span>Skills</span>
+    <span class="span-language">Skills</span>
     <img class="skillsArrow" src="assets/leftSideArrow.svg" alt="leftSideArrow">
   </div>
-  <hr>
+  <hr> 
+</div> 
 </section>
+
 <!--  ______________________________Footer-FORM-section___________________________  -->
-<footer>
+<footer class="footer-mobile footer-desktop">
   <section class="form-container">
```

Comment:
- [ ] For this section as requested by the previous reviewer kindly consider sticking to the Figma design as much as possible by adding the exact background image 
- your design 👇 
<img width="1408" alt="Screen Shot 2023-06-17 at 21 05 01" src="https://github.com/fmalgarini/My-portfolio-Project/assets/60146030/a3d28b01-673c-4d7e-8368-643fbd02a830">

- Figma design 👇 
<img width="789" alt="Screen Shot 2023-06-17 at 21 07 59" src="https://github.com/fmalgarini/My-portfolio-Project/assets/60146030/b532cc73-092a-4003-9a70-319f01cbc64b">